Modules/System Tests/URI/src/UriBuilderTest.Codeunit.al

```diff
@@ -22,6 +22,83 @@ codeunit 135071 "Uri Builder Test"
         Assert.ExpectedError('A call to System.UriBuilder failed with this message: Invalid URI');
     end;
 
+    [Test]
+    [Scope('OnPrem')]
+    procedure SetAndGetSchemeTest()
+    var
+        Uri: Codeunit Uri;
+    begin
+        // [Given] A URI with an http scheme
+        UriBuilder.Init('http://microsoft.com');
```

Comment:
Maybe assert the right scheme is in place after initialization.

`Assert.AreEqual('https', UriBuilder.Scheme(), 'The scheme does not match');`